ubuntudesign/www.ubuntu.com
# fix up /download/cloud

## Done

* removed old form from /download/cloud
* added download link directly to /download/cloud/install-autopilot-testdrive

## QA

1. go to /download/cloud and see that there is no more form
2. go to /download/cloud/install-autopilot-testdrive and see that the link 'Need the download' now downloads the binary

## Issue / Card

[List of links to Github issues/bugs and cards if needed]

[trello](https://trello.com/c/T3GFpR9w)
